src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/domain/PractitionerWorkload.kt

```diff
@@ -32,7 +32,11 @@ data class PractitionerWorkload @JsonCreator constructor(
         choosePractitionerResponse.name,
         tier,
         choosePractitionerResponse.probationStatus,
-        choosePractitionerResponse.communityPersonManager,
+        if (choosePractitionerResponse.communityPersonManager?.isUnallocated == true) {
```

Comment:
can this be simplified to `choosePractitionerResponse.communityPersonManager?.takeUnless { it.isUnallocated}` ?